.github/workflows/run-helm.yml

```diff
@@ -32,14 +32,38 @@ jobs:
           # pull_request_target is run in the context of the base repository
           # of the pull request, so the default ref is master branch and
           # ref should be manually set to the head of the PR
-          ref: refs/pull/${{ github.event.pull_request.number }}/head
+          ref: refs/pull/${{ github.event.pull_request.number }}/merge
+          # repository: ${{ github.event.pull_request.head.repo.full_name }}
+          # ref: ${{ github.event.pull_request.head.ref }}
 
       - name: Checkout the head commit of the branch
         if: ${{ github.event_name != 'pull_request_target' }}
         uses: actions/checkout@v2
 
       - uses: azure/setup-helm@v1
 
+      - name: foo
+        run: |
+          printenv
+          echo ${{ github.event_name }}
+
+      # We should generate two sets of akri charts.
+      # One for our releases (akri) and one for our merged PRs (akri-dev).  'akri' should only be created for
+      # our releases and should default to not use dev containers.  'akri-dev'
+      # should build for any merged PR and should default to use our dev containers.
+      - name: Split into akri and akri-dev charts (this step is for release == akri)
+        if: github.event_name == 'pull_request_target'
+        run: |
+          sed -i s/"name: akri"/"name: akri"/g ./deployment/helm/Chart.yaml
+          sed -i s/"description: A Helm chart for Akri"/"description: A Helm chart for Akri releases"/g ./deployment/helm/Chart.yaml
+          sed -i s/"useDevelopmentContainers: true"/"useDevelopmentContainers: false"/g ./deployment/helm/values.yaml
+      - name: Split into akri and akri-dev charts (this step is for !release == akri-dev)
```

Comment:
The condition here does not seem to be a negation of the earlier one, is that by design?

Reply:
in what sense ... one of them says event_name==pull_request_target and another says event_name!=pull_request_target?

Reply:
i think it probably needs changing anyways ...

release: event_name == 'release'
dev: event_name != 'release'

Reply:
!release==akri-dev
release==akri

(perhaps the first one should be release==akri-dev?)

Reply:
ah!  i see, you are right!  will fix.

Reply:
nit: I think the -name reference still needs fixing.

Reply:
the parenthetical is correct, right?

first one is for akri and release:
```yaml
      - name: Split into akri and akri-dev charts (this step is for release == akri)
        if: (github.event_name == 'release')
        run: |
          sed -i s/"name: akri"/"name: akri"/g ./deployment/helm/Chart.yaml
```

Second one is for akri-dev and !release:
```yaml
      - name: Split into akri and akri-dev charts (this step is for !release == akri-dev)
        if: (github.event_name != 'release')
        run: |
          sed -i s/"name: akri"/"name: akri-dev"/g ./deployment/helm/Chart.yaml
```

Reply:
thanks for the help!  tried to clarify the name.